### PR TITLE
Added models.py file and University class within it

### PR DESCRIPTION
**Pull request by DeanCivlin**

### CODE REVIEW

**Review by Yarboa — COMMENTED** [review]
Please add some methods to University
And than add one or two methods and test it with tests,

You are populating input from migration test file.
You can add a list and/or search method and test it.

Other wise, when tests are running, we have no way to verify that this PR work is really working


**Note**

look at PR tests, 
./AcadeMeData/models.py:3:1: F401 'django.conf.settings' imported but unused
./AcadeMeData/migrations/0001_initial.py:18:121: E501 line too long (148 > 120 characters)

tests should pass oi order to be merged"

**Note**
Add reviewers top right, first, you should add team/project members to review
and then the mentors

**Comment by DeanCivlin** [comment]
@Yarboa Please review my latest changes.
I know it is not done yet but it would be helpful to receive some guidelines on what and where should I improve the code.

File: AcadeMeData/tests.py [hunk 1]
```diff
@@ -1,3 +1,9 @@
-# from django.test import TestCase
+import pytest
+from AcadeMeData.models import University
 
-# Create your tests here.
+
+class TestUniversityModel(pytest):
+    @pytest
+    def test_get_university_by_id(self):
```
Comment (Yarboa) [inline AcadeMeData/tests.py:7]: Do you think this method makes sense ? 
I would consider searching university by name, for a start

File: AcadeMeData/models.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    name = models.CharField(max_length=100)
+    location = models.CharField(max_length=100)
+    # maybe change here to: description = models.TextField()
+    description = models.TextField(null=True, blank=True)
+
+    def __str__(self):
+        return self.name
+
+    @staticmethod
+    def get_university_by_id(university_id):
```
Comment (Yarboa) [inline AcadeMeData/models.py:17]: see my comment below, add filter to search university name and location
WDYT?

**Comment by DeanCivlin** [comment]
@Yarboa made changes according to your review. please take a look

**Review by DanielItkes — APPROVED** [review]

**Review by Yarboa — APPROVED** [review]
LGTM, good work

**Review by Yarboa — CHANGES_REQUESTED** [review]
Currently your test, or any test is not running in github,
Please refer how to add tests to workflow

run the following pipenv install --dev pytest-django
add pytest.ini
add .github/workflows/pytest.yml
see last class talk please

File: AcadeMeData/tests.py [hunk 3]
```diff
@@ -4,6 +4,13 @@
...
-        universityTest = University.get_university_by_id(1)
+    def test_get_university_by_name(self):
+        universityTest = University.get_university_by_name(
+            'Reichman University')
+        assert universityTest.exist()
+    
+    @pytest
+    def test_get_university_by_location(self):
+        universityTest = University.get_university_by_location(
+            'Jerusalem')
```
Comment (bregman-arie) [inline AcadeMeData/tests.py:15]: Please make Jerusalem an argument the function gets

**Review by bregman-arie — CHANGES_REQUESTED** [review]

**Review by Yarboa — COMMENTED** [review]
Please rebase the PR with upstream

**Comment by Yarboa** [comment]
> Currently your test, or any test is not running in github, Please refer how to add tests to workflow
> 
> run the following pipenv install --dev pytest-django add modified Pipfiles add pytest.ini add .github/workflows/pytest.yml see last class talk please

Already Merged, please rebase, verify your tests are runing

**Comment by DeanCivlin** [comment]
Code was merged to other PR. closed due to redundancy.

**Pull request closed without merging**